stac_model/stac_model/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""A PydanticV2 validation and serialization library for the STAC ML Model Extension"""
+
+from importlib import metadata
+
+try:
+    __version__ = metadata.version("stac-model")
+except metadata.PackageNotFoundError:
+    __version__ = "unknown"
```

Comment:
Ins't this supposed to be defined by this module? Doesn't `metadata.version()` look for this `__version__` once installed?

Reply:
metadata.version() gets the version from the pyproject.toml package metadata. That way we don't need to separately define the version in the .toml and then also update it here when we increment the version.

https://github.com/crim-ca/dlm-extension/blob/b2318ad47543db4428c0da4d908edbcdf4bec343/stac_model/pyproject.toml#L10

```
>>> import stac_model
>>> stac_model.__version__
'0.1.1a2'
```